Source/ARTDefault.h

```diff
@@ -37,6 +37,8 @@
 
 + (NSString *)libraryVersion;
 
++ (NSString *)userAgent;
```

Comment:
I would rename this to `ablyAgent`, or even just `agent`.

The `user` suffix suggests too strongly that this is a `User-Agent` as classically understood by most developers (i.e. HTTP context), whereas `User-Agent` was just used as inspiration for [RSC7d2](https://docs.ably.io/client-lib-development-guide/features/#RSC7d2).

Reply:
fixed

Reply:
https://github.com/ably/ably-cocoa/pull/1119/commits/103f979058b99abad6d497b83697638825f1710e